packages/shared/styles/components/SfHero.scss

```diff
@@ -0,0 +1,160 @@
+@import '../variables';
+@import '~@glidejs/glide/src/assets/sass/glide.core';
+
+$hero--mobile-breakpoint: $mobile-max;
+
+$hero__controls--top: 50%;
+$hero__controls--padding-h: 1rem;
+$hero__controls--width: calc(100% - 2 * #{$hero__controls--padding-h});
+
+$hero__bullets--position-offset: 0.5rem;
+$hero__bullets--position-offset-desktop: 2.5rem;
+$hero__bullets--width: 100%;
+
+$hero__item--height: 14rem;
+$hero__item--height-desktop: 30rem;
+$hero__item--width: 100%;
+$hero__item--padding: 1rem;
+$hero__item--padding-desktop: 5rem;
+$hero__item--bg-size: cover;
+$hero__item--bg-repeat: no-repeat;
+
+$hero-item__container--width-desktop: 50%;
+$hero-item__container--width: 60%;
+$hero-item__container--height: 100%;
+$hero-item__container--padding-desktop: 0 2rem;
+$hero-item__container--padding: 0;
+
+$hero-item__subtitle--width: 100%;
+$hero-item__subtitle--width-desktop: min-content;
+$hero-item__subtitle--color: $c-gray-secondary;
+$hero-item__subtitle--border-left: $hero-item__subtitle--color solid 1px;
+$hero-item__subtitle--border: 0;
+$hero-item__subtitle--font: 10px;
+$hero-item__subtitle--font-desktop: 12px;
+$hero-item__subtitle--line_height-desktop: 17px;
+$hero-item__subtitle--line_height: 14px;
+$hero-item__subtitle--padding_left-desktop: 36px;
+$hero-item__subtitle--padding_left: 18px;
+
+$hero-item__title--font: 18px;
+$hero-item__title--font-desktop: 36px;
+$hero-item__title--line-height-desktop: 50px;
+$hero-item__title--line-height: 25px;
+$hero-item__title--color: $c-dark-primary;
+
+.sf-hero{
+  position: relative;
+
+  &__slides{
+    margin: 0;
+  }
+
+  &__controls{
+    display: none;
+
+    @media screen and (min-width: $hero--mobile-breakpoint){
+      position: absolute;
+      top: $hero__controls--top;
+      left: 0;
+      transform: translate3d(0, -$hero__controls--top, 0);
+      width: $hero__controls--width;
+      display: flex;
+      justify-content: space-between;
+      padding: 0 $hero__controls--padding-h;
+    }
+  }
+
+  &__bullets{
+    @media screen and (min-width: $hero--mobile-breakpoint){
+      
+    }
+    position: absolute;
+    width: $hero__bullets--width;
+    display: flex;
+    justify-content: center;
+    align-content: center;
+
+    &--bottom {
+      bottom: $hero__bullets--position-offset;
+      @media screen and (min-width: $hero--mobile-breakpoint){
+        bottom: $hero__bullets--position-offset-desktop;
+      }
+    }
+    &--top {
+      top: $hero__bullets--position-offset;
+      @media screen and (min-width: $hero--mobile-breakpoint){
+        top: $hero__bullets--position-offset-desktop;
+      }
+    }
+  }
+
+  &-item {
```

Comment:
I discussed it with @filrak and we decided to treat <code>item</code> as a separate BEM block.
Then class name will be <code>.sf-hero-item__title</code> in stead of <code>.sf-hero__item__title</code>. In other words we merged <code>SfHeroItem.scss</code> into <code>SfHero.scss</code>, but kept naming convention as for separate files, to avoid double underscores.
Wdyt?